assets/xml/overlays/ovl_En_Sth2.xml

```diff
@@ -0,0 +1,14 @@
+<Root>gIceArrowMaskTex
+    <File Name="ovl_En_Sth2" BaseAddress="0x80BF74E0" RangeStart="0x460" RangeEnd="0x2B30">
+        <Texture Name="gEnSth2SkinBlotchTex" OutName="D_80BF7940" Format="rgba16" Width="8" Height="8" Offset="0x460"/>
```

Comment:
`OutName` is the name of the generated png. Since you named the textures you should name those with snake_case
```suggestion
        <Texture Name="gEnSth2SkinBlotchTex" OutName="skin_blotch" Format="rgba16" Width="8" Height="8" Offset="0x460"/>
```